main.py

```diff
@@ -0,0 +1,40 @@
+import requests
+from urllib.parse import urlparse 
+
+Apache_Type = ["text/plain", "text/plain; charset=ISO-8859-1", 
+                "text/plain; charset=iso-8859-1", 
+                "text/plain; charset=UTF-8"]
+
+check_for_apache = False
+
+#handling resource metadata
+def _get_resource_metadata(body, ContentType, http_origin):
+    if http_origin:
+        # resource retrieved via HTTP
+
+        # TODO (Handle multiple Contentype headers)
+        
+        if ContentType in Apache_Type:
+            check_for_apache = True
+        
+    return ContentType
+
+
+#main function
+def xtractmime(body, ContentType, http_origin, NoSniff):
+    supplied_type = _get_resource_metadata(body, ContentType, http_origin)
+
+    pass
+
+# SAMPLE INPUT
+if __name__ == '__main__':
```

Comment:
Seems like this block is only for testing. In that sense, I'd suggest moving the imports from the top of the file to the block, and moving the block itself to a different file (as a temporary solution anyway, the end goal would be to have a test suite in a separate directory).

Reply:
Yes, test suite will be in a separate directory, these are just temporary.  I will move to a different file